tools/lkl/fuzzers/binder/README.md

```diff
@@ -0,0 +1,84 @@
+# Android Binder fuzzer
```

Comment:
Thanks @rodionov @abcSup for the great documentation. Eventually I think we want to move this to using kernel .rst, I have create #566 to track this work.

Reply:
Thanks, Tavi! Happy to convert this readme.md to .rst in the subsequent work.